src/test/ui/error-trait/error-in-core-no-feature.rs

```diff
@@ -0,0 +1,18 @@
+// edition:2021
+#![crate_type = "lib"]
+#![no_std]
+
+use core::fmt;
+
+#[derive(Debug)]
+struct MyCoreError(());
+
+impl fmt::Display for MyCoreError {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        unimplemented!()
+    }
+}
+
+impl core::error::Error for MyCoreError {} //~ ERROR use of unstable library feature 'error_in_core'
```

Comment:
Turns out that the `unstable` attributes don't actually work the way I expected them to. I had expected this to be an error the same way the it is in the above test but it actually compiles just fine...